MegaSolution/CreatingLineFromRandomPartsOfOthers/Inputter.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace CreatingLineFromRandomPartsOfOthers
+{
+  class Inputter
+  {
+    const string FIRSTINSTRUCTION = "Enter the first line, please";
+    const string SECONDINSTRUCTION = "Enter the second line, please";
+    const string ASKFORREINPUT = "Pease, repeat your input";
+    bool successInput;
+    public string[] InputLines(string[] lines)
```

Comment:
why you pass lines in InputLines? You return lines from methods, this is wrong decision.  Create lines inside method